src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header,
+ul,
+a {
+  display: flex;
+}
+
+header {
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* Logo */
+img {
+  margin-left: 50px;
+}
+
+/* Nav */
+nav {
+  text-transform: uppercase;
+  margin-right: 50px;
+}
```

Comment:
Instead of `margin-left` and `margin-right` you can add `padding: 0 50px` to header` and `box-sizing: border-box` rule for `*` selector